src/data/faq_de.json

```diff
@@ -1134,6 +1134,7 @@
                         "anchor": "API10",
                         "active": true,
                         "textblock": [
+                            "<b>AKTUELL 14.10.2020</b><br/>Sollten Sie dieses Problem haben, würden wir es zu schätzen wissen, Ihren Bericht hier zu erhalten: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1962' target='_blank' rel='noopener'>cwa-app-android/issues/1962</a>.<hr/>",
```

Comment:
The date should be 14.01.2021 (today), not 14.10.2020.

Reply:
thanks